app/controllers/scorecard_player_builders_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ScorecardPlayerBuildersController < ApplicationController
+  respond_to :html
```

Comment:
Ah, this tells rails that every action in this controller is going to respond only to an html request (as opposed to other mime type requests like json or xml). 

By doing this, it allows you to use some different syntax. I thought I needed it, but it's totally unnecessary.

Line 2 has been removed!